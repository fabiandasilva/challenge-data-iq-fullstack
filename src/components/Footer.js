import React from "react";
import Image from "next/image";
import Github from "../../public/image/icons/github.svg";

const Footer = () => {
  return (
    <footer>
      <div className="flex items-center gap-2 bg-white h-20 justify-end px-5">
        <a href="https://github.com/fabiandasilva" target="_blank">
          <Image
            src={Github}
            width="20"
            height="20"
            alt="icon"
            className="h-5 w-5"
          />
        </a>
        <span className="text-zinc-700 cursor-pointer">
          <a href="https://www.linkedin.com/in/fabiandasilva/" target="_blank">
            Fabian Da Silva
          </a>
        </span>
      </div>
    </footer>
  );
};

export default Footer;
