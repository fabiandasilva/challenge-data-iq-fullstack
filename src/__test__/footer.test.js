/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import React from "react";
import Footer from "../components/Footer";

test("Footer se renderiza correctamente", () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
