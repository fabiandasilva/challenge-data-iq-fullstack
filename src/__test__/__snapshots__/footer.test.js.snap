// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer se renderiza correctamente 1`] = `
<DocumentFragment>
  <footer
    class="flex items-center gap-2 bg-white h-20 justify-end px-5"
  >
    <a
      href="https://github.com/fabiandasilva"
      target="_blank"
    >
      <img
        alt="icon"
        class="h-5 w-5"
        data-nimg="1"
        decoding="async"
        height="20"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=48&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=48&q=75 2x"
        style="color: transparent;"
        width="20"
      />
    </a>
    <span
      class="text-zinc-700 cursor-pointer"
    >
      <a
        href="https://www.linkedin.com/in/fabiandasilva/"
        target="_blank"
      >
        Fabian Da Silva
      </a>
    </span>
  </footer>
</DocumentFragment>
`;
